engine/runtime-compiler-dump-igv/src/main/java/org/enso/compiler/dump/igv/ASTNode.java

```diff
@@ -0,0 +1,144 @@
+package org.enso.compiler.dump.igv;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.enso.compiler.core.IR;
+import org.enso.compiler.core.ir.MetadataStorage;
+
+final class ASTNode {
```

Comment:
Why there are these wrappers? The dumping should work directly on `IR` and its elements, it shouldn't require any new allocations.

Reply:
As discussed on a call, it is not possible to use plain `IR` here. We need those wrappers.